changelog/fragments/2852-pr.yaml

```diff
@@ -0,0 +1,34 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. [#2852](https://github.com/operator-framework/operator-sdk/pull/2852)
+
+# kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "removal"
+
+    # Is this a breaking change?
+    breaking: true
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: The `inotify-tools` dependency was removed and `/bin/ao-logs` are no longer generated for Ansible based-operators
+      body: >
+        The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated in the previous versions was removed. This dependency was used to output the Ansible logs in a side-car container called `ansible`, which has no longer been scaffolded by default. In this way, if you have any customization using this dependency be aware that it is no longer in the Ansible based-operator image and then, feel free to remove this file `/bin/ao-logs` for your projects since it has no longer usage.  For further information also see the section `Remove Ansible container sidecar` in this guide.
```

Comment:
> For further information also see the section `Remove Ansible container sidecar` in this guide.

We're splitting out the migration guide into separate files for each release, so this text will end up in `v0.18.0.md`. I think we're also planning to split the existing migration guide into individual release migration guides, so linking over to that file might not be the best idea either.

Is there a PR related to removing the ansible container sidecar we could link to instead?

Reply:
For the users know the steps, I think that the best would be the migration guide section and not the PR. I will try to figure out something here.

Reply:
If this isn't urgent, I can try to throw together a PR to split out the existing version upgrade guide, get that merged, and then you can link to the new one?

Reply:
It has not the links. I mean it has just the information and then it was updated based on this comment. 
Could you please check and let us know if you are ok with?